scale/storage/fixtures/destroy_file_errors.json

```diff
@@ -0,0 +1,15 @@
+[
+    {
+        "model": "error.Error",
```

Comment:
I don't think we need this error. If the file doesn't exist, then we can consider the job successful.